auth-manager-app/src/components/Permissions/ChangePermissionsThunk.jsx

```diff
@@ -0,0 +1,82 @@
+import { createAsyncThunk } from '@reduxjs/toolkit';
+
+/**
+ * Adds a new policy to a certain user
+ * @param {string} authKey - the authentication key, from Login
+ * @param {string} username - the username of the user row being clicked, from Users
+ * @param {string} envir - the selected environment, from Login
+ * @param {string} policy_name - the policy name we want to add to the user
+ */
+export const addPolicy = createAsyncThunk(
+  'changePermissions/addPolicy',
+  async ({ authKey, username, envir, policy_name }, { rejectWithValue }) => {
```

Comment:
Naming nit: I feel `domain` would be a better fit to `envir` (here and everywhere it's used in the app) because it's more transparent that it's a domain name string and "environment" is a fairly overloaded term in the software world